docs/setup-cluster/security/tls.rst

```diff
@@ -75,3 +75,247 @@ untrusted certificate and display a hash of the certificate. You may wish to con
 your system administrator; in any case, if you confirm the connection to the master, the certificate
 will be stored on the computer where the CLI is being run and future connections to the master will
 be made without confirmation.
+
+*************************************
+ Let's Encrypt TLS Certificate Setup
+*************************************
+
+This section describes how to set up the Let's Encrypt TLS certificate.
+
+Installing a Let's Encrypt TLS certificate requires an ACME protocol client ``certbot`` and one of
+the following domain token verification methods:
+
+-  HTTP-01, or
+-  DNS-01
```

Comment:
```suggestion
This section describes how to set up a TLS certificate from `Let's Encrypt
<https://letsencrypt.org>`__ using `Certbot <https://certbot.eff.org/>`__ and either the HTTP-01 or
the DNS-01 challenge type.
```

Also, there really ought to actually be a section about using the HTTP-01 challenge, instead of only jumping to the behind-a-VPN scenario. And there should be something about how to pick which one to use (which is sort of implicitly described below, but it's buried instead of right here at the top).

Reply:
@ioga or @dzhu please provide the content that you want added.

Reply:
I am okay with the existing content...